packages/ontario-frontend-cli/create/templates/package.njk

```diff
@@ -1,7 +1,7 @@
 {
   "name": "{{ projectName }}",
   "version": "0.0.1",
-  "description": "{{ projectDescription }}",
+  "description": "Ontario Frontend Project (Add your project description here)",
```

Comment:
Same comment as the one in the README template. Can we make this a bit more of a call-out?

For Example:
`Ontario Frontend Project (TODO: Replace with a description of your project)`